Integrations/GoogleKeyManagementService/GoogleKeyManagementService.py

```diff
@@ -0,0 +1,956 @@
+            }, asymmetric_encrypt_context)
+
+
+def asymmetric_decrypt_command(client: Client, args: Dict[str, Any]) -> Tuple[str, Any, Any]:
+    """Decrypt chipertext to plaintext using asymmetric key.
+
+    Args:
+        client(Client): User's Client.
+        args(dict): Demisto arguments.
+
+    Returns:
+        The decrypted plaintext.
+    """
+    project_id, location_id, key_ring_id, crypto_key_id = demisto_args_extract(client, args)
+    crypto_key_version = args.get('crypto_key_version')
+
+    # Construct the resource name of the CryptoKeyVersion.
+    crypto_key_version_name = client.kms_client.crypto_key_version_path(project_id, location_id, key_ring_id,
+                                                                        crypto_key_id, crypto_key_version)
+
+    ciphertext = base64.b64decode(str(args.get('ciphertext')))
+
+    response = client.kms_client.asymmetric_decrypt(crypto_key_version_name, ciphertext)
+
+    # handle the created plaintext back to base64 if needed and clear added characters.
+    if args.get('is_base64') == 'true':
+        plaintext = str(base64.b64encode(response.plaintext))[2:-1]
+
+    else:
+        plaintext = str(base64.b64decode(response.plaintext))[2:-1]
+
+    asymmetric_decrypt_context = {
+        'CryptoKey': crypto_key_id,
+        'IsBase64': args.get('is_base64') == 'true',
+        'Plaintext': plaintext
+    }
+
+    return (f"The text has been decrypted.\nPlaintext: {plaintext}",
+            {
+                f'{INTEGRATION_CONTEXT_NAME}.AsymmetricDecrypt(val.CryptoKey == obj.CryptoKey '
+                f'&& val.IsBase64 == obj.IsBase64 && val.Plaintext == obj.Plaintext)': asymmetric_decrypt_context,
+            }, asymmetric_decrypt_context)
+
+
+def test_function(client: Client) -> None:
+    """Test's user's input:
+        for Encrypter, Decrypter, Encrypter/Decrypter Roles - checks Service account json formatting only.
+        for Project-Admin and KMS-Admin Roles - checks the Service account json formatting and connection to project.
+
+    Args:
+        client(Client): User Client.
+    """
+    # if the user role is Encrypt/Decrypt related then no additional checks are possible.
+    if client.role in ['Project-Admin', 'KMS-Admin']:
+        # listing the KeyRings in order to check responses from the API.
+        location_path = client.kms_client.location_path(client.project, client.location)
+        # the response is a iterable containing the KeyRings info.
+        response = client.kms_client.list_key_rings(location_path)
+
+        # Count the number of existing KeyRings.
+        if not list(response):
+            raise Exception("No response - please check Project in Google Cloud KMS and Default Location")
+
+
+def main():
+    COMMANDS = {
+        f'{INTEGRATION_COMMAND_NAME}create-key': (create_crypto_key_command, ['Project-Admin', 'KMS-Admin']),
```

Comment:
It will look nicer to store it this way, and update the CONST accrodingly
```suggestion
        f'{INTEGRATION_COMMAND_NAME}-create-key': (create_crypto_key_command, ['Project-Admin', 'KMS-Admin']),
```

Reply:
will do